### DISCUSSION TITLE
Have Grand Prix radio only show for Dutch speaking viewers

### DISCUSSION THREAD

**Discussion by BeanXOfficial** [f1multiviewer/issue-tracker#34] [Feature requests]
It would be a small change but might just clean the UI up a little bit. There could be a window asking "What language do you prefer" first time starting the app. Based on the response it would only show grand prix radio to the people that prefer Dutch. And maybe a little country flag in the top right so you can change it in case you change your mind.

Maybe this could also be used to automatically set the stream default language

**Comment by ManCaveMedia** [comment]
Or add it to the settings menu, so you can enable it/choose the preferred language from there.

**Reply by maartenzuidland** [reply]
My preference in all applications and OS is set to English, but I still prefer GP Radio as my F1 commentary.
Putting it in Settings would make it available when all other settings would point to not showing the option if you try to “guess” it.
